packages/flutter/lib/src/widgets/app.dart

```diff
@@ -0,0 +1,223 @@
+// Copyright 2015 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:ui' as ui show Locale, WindowPadding, window;
+
+import 'package:flutter/rendering.dart';
+import 'package:flutter/services.dart';
+
+import 'asset_vendor.dart';
+import 'basic.dart';
+import 'binding.dart';
+import 'checked_mode_banner.dart';
+import 'framework.dart';
+import 'locale_query.dart';
+import 'media_query.dart';
+import 'navigator.dart';
+import 'performance_overlay.dart';
+import 'semantics_debugger.dart';
+import 'title.dart';
+
+AssetBundle _initDefaultBundle() {
+  if (rootBundle != null)
+    return rootBundle;
+  return new NetworkAssetBundle(Uri.base);
+}
+
+final AssetBundle _defaultBundle = _initDefaultBundle();
+
+class RouteArguments {
+  const RouteArguments({ this.context });
+  final BuildContext context;
+}
+typedef Widget RouteBuilder(RouteArguments args);
+
+typedef Future<LocaleQueryData> LocaleChangedCallback(Locale locale);
+
+class WidgetsApp extends StatefulComponent {
+  WidgetsApp({
+    Key key,
+    this.title,
+    this.textStyle,
+    this.color,
+    this.routes: const <String, RouteBuilder>{},
+    this.onGenerateRoute,
+    this.onLocaleChanged,
+    this.showPerformanceOverlay: false,
+    this.showSemanticsDebugger: false,
+    this.debugShowCheckedModeBanner: true
+  }) : super(key: key) {
+    assert(routes != null);
+    assert(routes.containsKey(Navigator.defaultRouteName) || onGenerateRoute != null);
+    assert(showPerformanceOverlay != null);
+    assert(showSemanticsDebugger != null);
+  }
+
+  /// A one-line description of this app for use in the window manager.
+  final String title;
+
+  /// The default text style for [Text] in the application.
+  final TextStyle textStyle;
+
+  /// The accent color to use for the application in the operating system
```

Comment:
accent  <--- not sure if this term makes sense here.  Maybe just drop this word.